integration_tests/tests/image-repository.scenario.ts

```diff
@@ -0,0 +1,82 @@
+import { browser, ExpectedConditions as until, $, $$ } from 'protractor';
+import { appHost, registryHost, testName, userName, userPasswd } from '../protractor.conf';
+import * as loginView from '../views/quay-login.view';
+import * as createrepoView from '../views/image-repository.view';
+import {execSync} from 'child_process';
+
+const BROWSER_TIMEOUT = 15000;
+
+describe('Create an image repository', () => {
+  it(`create image repository ${testName} if necessary`, async() => {
+    await createrepoView.createrepo(testName, 1);
+    await browser.sleep(1000);
+    expect(browser.getCurrentUrl()).toContain(testName);
+  });
+});
+
+describe('Push/pull images to/from Quay via podman', () => {
+  it(`push image to repository ${testName}`, async() => {
+    execSync(`sudo podman pull docker.io/busybox`);
+    execSync(`sudo podman tag docker.io/busybox ${registryHost}/${userName}/${testName}:busybox`);
+    execSync(`sudo podman push --tls-verify=false --creds=${userName}:${userPasswd} ${registryHost}/${userName}/${testName}:busybox`);
+    await browser.get(`${appHost}/repository/${userName}/${testName}?tab=tags`);
+    await browser.sleep(1000);
+    expect(createrepoView.tags.getText()).toBe('busybox');
+  });
+
+  it(`pull image to repository ${testName}`, async() => {
+    execSync(`sudo podman rmi ${registryHost}/${userName}/${testName}:busybox`);
+    execSync(`sudo podman pull --tls-verify=false ${registryHost}/${userName}/${testName}:busybox`);
+  });
+});
+
+describe('Viewing and modifying tags', () => {
+  beforeAll(async() => {
+    await browser.get(`${appHost}/repository/${userName}/${testName}?tab=tags`);
+    await browser.sleep(1000);
+  });
+
+/*  it('add new tag for image manifest', async() => {
```

Comment:
We shouldn't have commented-out code. Is there something wrong with these tests?

Reply:
Because it is unstable due to locate elements by xpath, I want to improve this part in the future

Reply:
You can skip these tests [using `xit(...)`](https://jasmine.github.io/api/3.5/global.html#xit) instead of commenting out the code.

Reply:
Please use `xit()` instead of commented-out code.